### PR TITLE
Add breaking change "Clip behavior"

### PR DESCRIPTION
**Pull request by liyuqian**

### CODE REVIEW

File: src/docs/release/breaking-changes/clip-behavior.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+3.  Add `clipBehavior: Clip.antiAlias` if you need anti-aliased clipping. This will give you smoother edges at a slightly higher cost. This will be the common case when dealing with circles and arcs.
+4.  Add `clip.antiAliasWithSaveLayer` if you want the exact same behavior as before (May 2018). Be aware that it’s very costly in performance. This will be only rarely needed. One case where you might need this is if you have an image overlaid on a very different background color. In these cases, consider whether you can avoid overlapping multiple colors in one spot (e.g. by having the background color only present where the image is absent).
+
+For Stack widget specifically, if you previously used `overflow: Overflow.visible`, you shall simply remove `overflow: Overflow.visible` as now we default to not clip. If you used `overflow: Overflow.clip` or relied on the default `overflow: Overflow.clip` setup, you can now replace `overflow: Overflow.clip` with `clipBehavior: Clip.hardEdge`.
```
Comment (goderbauer) [inline src/docs/release/breaking-changes/clip-behavior.md:48]: I thought we decided to default the stack to hardEdges?

Comment (liyuqian) [reply src/docs/release/breaking-changes/clip-behavior.md]: Yes, thanks! Updated.

**Comment by sfshaza2** [comment]
@kwalrath, we can't land any breaking changes until the "Timeline" section lists a specific release. We are waiting for that on this lab.

**Comment by sfshaza2** [comment]
@liyuqian, where do we stand on this feature? Has it been merged yet?

**Comment by sfshaza2** [comment]
@liyuqian, I just looked this internal CL up and it appears to be deleted?

**Comment by liyuqian** [comment]
Yes. The latest CL would now be cl/333841120. It seems that the last Google roll was successful so I'm going to merge this soon.

**Pull request merged**